lib/api/include/irods/check_auth_credentials.h

```diff
@@ -0,0 +1,137 @@
+#ifndef IRODS_CHECK_AUTH_CREDENTIALS_H
+#define IRODS_CHECK_AUTH_CREDENTIALS_H
+
+/// \file
+
+struct RcComm;
+
+/// The input data type used to invoke #rc_check_auth_credentials.
+///
+/// \since 4.3.1
+typedef struct CheckAuthCredentialsInput // NOLINT(modernize-use-using)
+{
+    // NOLINTBEGIN(cppcoreguidelines-avoid-c-arrays, modernize-avoid-c-arrays)
```

Comment:
maybe a note about how this is currently only for native

Reply:
The API endpoint's documentation captures that information.